packages/rsdd/rsdd.0.1.0/opam

```diff
@@ -0,0 +1,45 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Bindings for RSDD"
+description:
+  "OCaml bindings for the Rust Decision Diagram (RSDD) library, generated with ocaml-rs."
+maintainer: ["Matt Wang <mxw@cs.washington.edu>"]
+authors: [
+  "Steven Holtzen <s.holtzen@northeastern.edu>"
+  "Matt Wang <mxw@cs.washington.edu>"
+]
+license: "MIT"
+homepage: "https://github.com/neuppl/rsdd-ocaml"
+bug-reports: "https://github.com/neuppl/rsdd-ocaml/issues"
+depends: [
+  "ocaml" {>= "4.08.0" & < "5"}
+  "dune" {>= "2.7" & >= "3.0"}
+  "conf-rust-2021"
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/neuppl/rsdd-ocaml.git"
+url {
+  src:
+    "https://github.com/neuppl/rsdd-ocaml/archive/refs/tags/0.1.0-alpha.tar.gz"
+  checksum: [
+    "md5=83caf7c551b6cfb023338f080550a56f"
+    "sha512=83a514a9b0b7d139597b9eb0afe85a8a86b20929390f192ad9c7954d269a773cea69b964ac0eca04989ffaa43b851bb68b90442a0ce1347f08ef805f5c26d0e1"
+  ]
+}
+x-ci-accept-failures: [
+	  "freebsd (experimental)" # cargo (and thus conf-rust-2021) is not available
```

Comment:
```suggestion
	  "freebsd" # cargo (and thus conf-rust-2021) is not available
```

Reply:
Also for your information, you can have a [`rsdd.opam.template` file](https://dune.readthedocs.io/en/stable/reference/packages.html#:~:text=If%20a%20pkg.opam.template%20file%20is%20found) in your repository to have Dune generated opam file contain these fields from the beginning (so to avoid the hassle to add them manually for every release)

Reply:
Thanks! I had tried just using `"freebsd"` earlier and it still reported an `X` in the CI run; for the commit that I just force-pushed with `opam publish`, the [CI is still reporting the same error](https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/46550be36bb9867df897692b8913e55aa2349835) (ex: [freebsd on 4.14](https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/46550be36bb9867df897692b8913e55aa2349835/variant/freebsd,freebsd-ocaml-4.14-amd64,rsdd.0.1.0)).